brightcove/brightcove-popup/src/components/gridLayout.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import Loader from "./loader";
+import { Layout } from "../model/layout.model";
+
+const GridLayout: React.FC<Layout> = props => {
+    const { isSelected, selectedVideoList, videos, totalVideos, checkFiles, handleSelect, loadContent } = props;
+    const renderVideos = isSelected ? selectedVideoList : videos;
+
+    return (
+        <ul className="grid-layout">
+            <div className="grid-body">
+                {renderVideos.length > 0 ? (
```

Comment:
No need to check length is greater than zero like `renderVideos.length > 0` 
You can directly check `renderVideos.length ? () : () ` that also works
`renderVideos.length` return true if array is not blank